about.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Africa Women Software Engineers</title>
+      <link rel="stylesheet" href="./CSS/about.css">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <header class="header">
+            <div class="media">
+                <ul class="media-nav d-flex flex-center">
+                    <li class="media-link">
+                        <a href="https://m.facebook.com/afrofemcoders/"><i class="fa fa-facebook"></i></a>
+                    </li>
+                    <li class="media-link">
+                        <a href="https://twitter.com/CynthiaInga_C"><i class="fa fa-twitter"></i></a>
+                    </li>
+                    <li class="media-link"><a href="#">English</a></li>
+                    <li class="media-link"><a href="#">My page</a></li>
+                    <li class="media-link"><a href="#">Logout</a></li>
+                </ul>
+            </div>
+            <nav class="nav-bar">
+                <div class="Hamburger d-flex flex-cente">
+                    <img src="./images/Icons/Hamburger_icon.svg.png" class="show-menu-btn" alt="Show menu">
+                    <img src="./images/Icons/Icon-Cancel.png" class="close-menu-btn" alt="close menu">
+                    <div class="logo">
+                        <img src="./images/Icons/Afrofemcode-logo.jpg" alt="logo">
+                    </div>
```

Comment:
[OPTIONAL]
- I would recommend changing the logo to improve your design and make it more professional. the background color of the logo is white color which doesn't fit the background color of the header.
Your logo:
<img width="143" alt="image" src="https://user-images.githubusercontent.com/77190203/152526315-97b523eb-316d-4437-bb2c-14ea9c9f91cc.png">

Reply:
Yeah, you are right, I changed the background color and put it white because I didn't find the logo that has the same background-color of my nav-bar.